frontend/src/util/tooltipLanguage.js

```diff
@@ -0,0 +1,24 @@
+export const tooltipLanguage = (stopCause) =>
+  ({
+    Average: 'REPLACE ME',
```

Comment:
Waiting to hear about this language that wasn't in the basecamp post: https://3.basecamp.com/3120230/buckets/18119898/messages/3864796186

Reply:
I'd also be okay setting this to `''` and adding average later